txstratum/manager.py

```diff
@@ -37,10 +38,15 @@ class TxMiningManager:
     BLOCK_TEMPLATE_UPDATE_INTERVAL = 3.0  # seconds
     TX_CLEAN_UP_INTERVAL = 300.0  # seconds
 
-    def __init__(self, backend: 'HathorClient'):
+    def __init__(self, backend: 'HathorClient', address: str):
         """Init TxMiningManager with backend."""
+        if address:
+            # Validate address. If the address is invalid, it raises an InvalidAddress exception.
+            decode_address(address)
```

Comment:
My previous comment goes for here as well.

Reply:
Fixed. Can you review again, please?

Reply:
You told me it's good in Slack.